static/routing-tree.js

```diff
@@ -225,7 +228,25 @@ function update(root) {
       .attr("dy", ".31em")
       .attr("text-anchor", function(d) { return d.x < 180 ? "start" : "end"; })
       .attr("transform", function(d) { return d.x < 180 ? "translate(8)" : "rotate(180)translate(-8)"; })
-      .text(function(d) { return d.receiver; });
+      .text(function(d) { return d.receiver; })
+      .on("mouseover", function(d) {
+        d3.select(this).style("fill", color);
+
+        text = jsyaml.dump(d.receiver_configs).replace(/ /g, '\u00a0').split("\n");
```

Comment:
Would be nice to detect `undefined` receiver configs here and not try to render them as YAML, but as `<receiver config missing>` or something like that.

Reply:
I've added your suggestion in the latest commit